docs/setup-guide.md

```diff
@@ -40,8 +40,12 @@ requirements:
 
  * Each node requires that the `mount.glusterfs` command is available. Under
   all Red Hat-based OSes this command is provided by the `glusterfs-fuse`
-  package.
-  
+  package. On Ubuntu, use the gluster PPA instead of the default repo:
+  ```
+  $ add-apt-repository ppa:gluster/glusterfs-3.12
```

Comment:
Use the 3.13 PPA instead.

Reply:
I had success with version 3.8.4 on CentOS 7.

Reply:
This should now be the 4.1 PPA :-)

Reply:
I agree. I reported this ages ago.